.github/workflows/build.yaml

```
@@ -114,7 +114,18 @@ jobs:
       date: ${{ inputs.date }}
       package-name: cugraph-pyg
       package-type: python
+  wheel-build-libwholegraph:
+    secrets: inherit
+    uses: rapidsai/shared-workflows/.github/workflows/wheels-build.yaml@branch-25.06
+    with:
+      build_type: pull-request
+      script: ci/build_wheel_libwholegraph.sh
+      package-type: python
```

This is a "cpp" wheel. xref libcudf wheels here: https://github.com/rapidsai/cudf/blob/b259cd58a5b1086c72465f6e2aaafe7ce5973a7d/.github/workflows/build.yaml#L96
```suggestion
      package-type: cpp
```